services/app-api/serverless.yml

```diff
@@ -64,6 +56,13 @@ custom:
       '/configuration/default/vpc/subnets/private/a/id': 'offline'
       '/configuration/default/vpc/subnets/private/b/id': 'offline'
       '/configuration/default/vpc/subnets/private/c/id': 'offline'
+      '/configuration/iam/full_permissions_boundary_policy': 'arn:aws:iam::local:policy/local/developer-boundary-policy'
+      '/configuration/email/sourceAddress': ${env:SES_SOURCE_EMAIL_ADDRESS, 'offline'}
```

Comment:
Is there a practical difference between setting the serverless-offline-ssm with an env fallback like sourceEmailAddress and setting a variable like reactAppOtelCollectorUrl above with an env var with a ssm fallback?

Reply:
They work the same really. I started with lookup for an env var since that's what we had been doing, and then a lookup to ssm if nothing was in the env var. We could just as easily do it the other way around if we want, I don't have a strong opinion here.

For the email config things I wanted to make it so we wouldn't need to change anything for local dev. We always pull the value from ssm, but locally that ssm value is set from `.envrc`.

Reply:
Ok, tickles my One Way To Do Things meter that we have two different ways to do this, but I think we need the serverless-offline-ssm plugin for the per-state-email-config for now anyway so it’s not like changing this would save us from this dependecny. I don’t think we’ll run into trouble with confusion over how to do this.